design/txn_author_agreement.md

```diff
@@ -83,27 +83,27 @@ Add new version of AML.
 Fetch AML from the ledger valid for specified time or the latest one.
 
 ### TAA Verification
-If TAA enabled on the ledger, then each write request from the user must contain metadata about a TAA signed by the user. The new format of write request is 
+If TAA enabled on the ledger, then each write request from the user must contain TAA acceptance signed by the user. The new format of write request is
 ```json=
-{ 
+{
     "reqId": INT,
     "operation" { ... },
     "identifier": "<str ident>",
     "protocolVersion": INT,
-    "signature": "<signature (txnAuthrAgrmtMeta is also signed)>",
-    "txnAuthrAgrmtMeta": { 
-        "acceptanceMechanismType": "label of type", 
-        "hash": "hash string", 
-        "timeOfAcceptance": <integer UTC TS>
+    "signature": "<signature (taaAcceptance is also signed)>",
+    "taaAcceptance": {
```

Comment:
This JSON is a part of public API. I don't like the idea to use abbreviation in public API here

Reply:
I don't mind calling Transaction Author Agreement just TAA. 
1) This is not true public API, at least not for the app developer. 
2) If the user doesn't understand what Transaction Author Agreement is, then it doesn't matter how we call it. Otherwise both terms will be clear, especially if we document it properly.
3) The size of request matters.

Reply:
When considering this design `TAA` abbreviation comes up numerous times, furthermore - we have request format documentation, where it can be (and should be) described as well.